utils/config/config.go

```diff
@@ -233,29 +233,52 @@ func saveConfig(config *ConfigV3) error {
 	return nil
 }
 
-func readConf() (*ConfigV3, error) {
+func readConf() (*ConfigV4, error) {
 	confFilePath, err := getConfFilePath()
 	if err != nil {
 		return nil, err
 	}
-	config := new(ConfigV3)
 	exists, err := fileutils.IsFileExists(confFilePath, false)
 	if err != nil {
 		return nil, err
 	}
-	if !exists {
-		return config, nil
-	}
-	content, err := fileutils.ReadFile(confFilePath)
-	if err != nil {
-		return nil, err
-	}
-	if len(content) == 0 {
-		return new(ConfigV3), nil
-	}
-	content, err = convertIfNeeded(content)
-	if err != nil {
-		return nil, err
+	config := new(ConfigV4)
```

Comment:
I think that the code of this function can be split into two functions, to make it easier to read.
For example, this function can call a new function, which returns the file content. The new function can tale care of finding the file.